### DISCUSSION TITLE
Customize the remark-lint style checker?

### DISCUSSION THREAD

**Discussion by burns-brian** [Zettlr/Zettlr#4828] [Q&A]
Is there a way to customize the remark-lint style checker? Our company had different style guidelines for certain elements, and it would be nice to customize the remark-lint/style checker.

**Comment by nathanlesage** [comment]
This will come – one of the main reasons for this not yet being possible is the now existing PR #4805 which will finally give me enough space to drop an entire phalanx of settings in there for that.

(Also, I will have to think about how to do it in a proper, user-friendly way, so if you have suggestions, keep 'em coming!)

**Reply by burns-brian** [reply]
Thanks for answering so quickly! No worries. Honestly, I 100% understand how complicated things can be. Even the simplest things can be so complicated, right? :-)

Just a suggestion, but it would be cool, even before there is a nice user-friendly interface, to be able to drop something equivalent to an .eslintrc.js file in a specific location. That might allow more tech-savvy people to test out features too. Of course I say that as someone who is a developer and graphic designer (i.e. someone who loves nice pretty design, which you obviously have). :-)

For my current purposes, I have several staff members who need to write Markdown the same way. Therefore, it would be great to drop some kind of .rc lint file for them. Actually, it could even exist in the workspace folder, as "rc" files are usually hidden anyway.
